src/components/Invoice.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from "react";
+import "../index.css";
+
+const Invoice = ({ invoice }) => {
```

Comment:
As you know, in Javascript there are several ways to define a function. For your App component, you've used `function App ()`, but for all your other components you've used the fat arrow syntax: `const ComponentName () =>`. It doesn't matter so much which one you use, so long as you pick one and stick with it to be consistent! (My preference here would be to use `function ComponentName ()`